### PR TITLE
Added option to estimate physical extension of objects.

### PR DESCRIPTION
**Pull request by lonbar**

### CODE REVIEW

File: background.py [hunk 1]
```diff
@@ -241,3 +241,21 @@ def hubble_distance(H0=constants['Hubble0']):
...
+    angular separation θ, 
+    redshift z, 
+    Hubble constant H0, 
+    energy densities ΩM, ΩDE, ΩR
+    EOS parameters w0, wa
+
+    Returns: spatial separation in kpc
+    '''
+
+    return 1e-3 * np.tan(θ) * angular_diameter_distance(z, H0, ΩM, ΩDE, ΩR, w0, wa)
```
Comment (marcobonici) [inline background.py:261]: Shouldn't the multiplicative factor `1e3` ? The angular diameter distance is given in Mpc.
```suggestion
    return 1e3 * np.tan(θ) * angular_diameter_distance(z, H0, ΩM, ΩDE, ΩR, w0, wa)
```

File: page_distances.py [hunk 2]
```diff
@@ -47,6 +47,7 @@ def show_page():
         st.write('Angular diameter distance at redshift', z_value, 'is:', str(round(bg.angular_diameter_distance(float(z_value), H0=float(H0_value), ΩM=float(ΩM_value), ΩDE=float(ΩDE_value), ΩR=float(ΩR_value), w0=float(w0_value) , wa=float(wa_value) ), sig_digits)), 'Mpc')
         st.write('Comoving volume at redshift', z_value, 'is:', str(round(1e-9*bg.comoving_volume(float(z_value), H0=float(H0_value), ΩM=float(ΩM_value), ΩDE=float(ΩDE_value), ΩR=float(ΩR_value), w0=float(w0_value) , wa=float(wa_value) ), sig_digits)), 'Gpc³')
         st.write('Lookback time at redshift', z_value, 'is:', str(round(bg.lookback_time(float(z_value), H0=float(H0_value), ΩM=float(ΩM_value), ΩDE=float(ΩDE_value), ΩR=float(ΩR_value), w0=float(w0_value) , wa=float(wa_value) ), sig_digits)), 'Gyrs')
+        st.write('An object spanning an angle of 1 arcsec at redshift', z_value, 'is', str(round(bg.proper_separation(180/3600/np.pi, float(z_value), H0=float(H0_value), ΩM=float(ΩM_value), ΩDE=float(ΩDE_value), ΩR=float(ΩR_value), w0=float(w0_value) , wa=float(wa_value)), sig_digits)), 'kpc across.')
```
Comment (marcobonici) [inline page_distances.py:50]: Maybe this comment is more general than the scope of this PR, but these sentences should be ordered in a better way.
For instance, I would move the new sentence just after the one about the angular diameter distance.

**Review by marcobonici — COMMENTED** [review]
Hi @lonbar ,
thank you for your PR.

I have just a couple of small changes I suggest, one being more quantitative and one more about style.
Since I am a bit tired, I may be wrong :smile: .

File: page_distances.py (hunk 2 above)
Comment (lonbar) [reply page_distances.py:50]: That makes sense, but I'd change that after the merge. What's important here is that all the information is correct, I think.

Comment (lonbar) [reply page_distances.py:50]: On second thought, let me change it now.

**Comment by lonbar** [comment]
Looking at the results section of the distance measures, I do see that there is some finetuning to be done. Having said that, I do think that this is somewhat outside of the scope of this MR. I'll create a separate branch for it. Agreed, @marcobonici?

**Comment by marcobonici** [comment]
Agreed!

**Pull request merged**